README.md

```diff
@@ -165,6 +165,13 @@ Results in the following JSON Schema files:
 
 </details>
 
+### Options
+
+The JSON Schema plugin supports the following options:
+
+- `additional_properties` - If specified, the generated schema will allow additional properties. Used to
```

Comment:
- What are additional properties?
- Why would I want to set them? I don't see how it relates to a client/sender
- Typically, options for protoc plugins are `key=value`, for bool it would be `additional_properties=true`

Reply:
This is json schema form the "ignore unknown fields" option from protojson: https://protobuf.dev/programming-guides/json/#json-options
update PR description and README.